roles/openshift_master/tasks/main.yml

```diff
@@ -56,7 +56,7 @@
       embedded_dns: "{{ openshift_master_embedded_dns | default(None) }}"
       dns_port: "{{ openshift_master_dns_port | default(None) }}"
       bind_addr: "{{ openshift_master_bind_addr | default(None) }}"
-      pod_eviction_timeout: "{{ openshift_master_pod_eviction_timeout | default(None) }}"
+      pod_eviction_timeout: "{{ lookup('oo_option', 'openshift_master_pod_eviction_timeout') | default(openshift_master_pod_eviction_timeout, true) | default(None) }}"
```

Comment:
This should probably be:
```
pod_eviction_timeout: "{{ lookup('oo_option', 'openshift_master_pod_eviction_timeout') | default(openshift_master_pod_eviction_timeout | default(None), true) }}"
```

One thing I think I would like to see though, is not pushing the oo_option filter down into the roles. 

Instead, you should be able to add the following to https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-master/config.yml#L19 for the same effect:
```
- set_fact:
    openshift_master_pod_eviction_timeout: {{ lookup('oo_option', 'openshift_master_pod_eviction_timeout') | default(None)
    when: openshift_master_pod_eviction_timeout is not defined
```